DFE-Digital/Dfe.Academies.External
# 104661 changes to a trust page

Resolves #<1234>

## Type of change

## Expected behaviour
There is a new page in the trust journey, step 2 of 3, to capture changes to a trust

## Steps to reproduce issue (if relevant)
n/a

## Steps to test this PR
1. go the trust area of application section 
2. get to step two in the trust journey
3. select either yes/no/unknown from the options
4. if yes is select an explanation will be required
5. select save and continue and you should be taken to the next stage in the journey

## Prerequisites
n/a
